src/flicking.js

```diff
@@ -1189,24 +1202,73 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 		},
 
 		/**
+		 * Update panel's previewPadding size according options.previewPadding
+		 */
+		_checkPadding: function() {
+			var options = this.options;
+			var previewPadding = options.previewPadding;
+			var padding = this.$wrapper.css("padding").split(" ");
+
+			// get current padding value
+			if (options.horizontal) {
+				padding = padding.length === 2 ?
+					[ padding[1], padding[1] ] : [ padding[3], padding[1] ];
+			} else {
+				padding = padding.length === 2 ?
+					[ padding[0], padding[0] ] : [ padding[0], padding[2] ];
+			}
+
+			padding = $.map(padding, function(num) {
+				return parseInt(num, 10);
+			});
+
+			if (previewPadding.length === 2 &&
+				previewPadding[0] !== padding[0] || previewPadding[1] !== padding[1]) {
+
+				this._setPadding(previewPadding);
+			}
+		},
+
+		/**
 		 * Update panel size according current viewport
 		 * @ko 패널 사이즈 정보를 갱신한다.
 		 * @method eg.Flicking#resize
+		 * @param {Boolean} checkPadding to check padding value
 		 * @return {eg.Flicking} instance of itself<ko>자신의 인스턴스</ko>
+		 * @example
+			var some = new eg.Flicking("#mflick", {
+				previewPadding: [10,10]
+			});
+
+			// when device orientaion changes
+			some.resize();
+
+			// when changes previewPadding option from its original value
+			some.options.previewPadding = [20, 30];
+			some.resize(true);
 		 */
-		resize: function () {
+		resize: function (checkPadding) {

```

Comment:
Adding a bool parameter to an existing function makes code hard to read without the documentation.
http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html


```
some.resize(true);
```
```
some.resize({ checkPadding: true });  // <-- easier to guess
```
How about using object parameter instead?